docs/en/guide/quota.md

```diff
@@ -4,29 +4,29 @@ sidebar_position: 5
 ---
 # Storage Quota
 
-JuiceFS v0.14.2 begins to support file system level storage quotas, a feature that includes:
+Since JuiceFS v0.14.2, JuiceFS have supported file system level storage quotas. This feature allows users to:
 
 - Limit the total available capacity of the file system
 - Limit the total inodes of the file system
 
 :::tip
-The storage quota settings are stored in the metadata engine for all mount points to read, and the client of each mount point will also cache its own used capacity and inodes and synchronize them with the metadata engine once per second, while the client will read the latest usage value from the metadata engine every 10 seconds to synchronize the usage information among each mount point, but this information synchronization mechanism does not guarantee that the usage data will be counted accurately.
+The storage quota settings are stored in the metadata engine for all mount points to read, and the client of each mount point will also cache its own used capacity and inodes and synchronize them with the metadata engine once per second. Meanwhile the client will read the latest usage value from the metadata engine every 10 seconds to synchronize the usage information among each mount point, but this information synchronization mechanism cannot guarantee that the usage data is counted accurately.
 :::
 
 ## View file system information
 
-In a Linux environment, for example, the default capacity of a JuiceFS type file system is identified as `1.0P` using the `df` command that comes with the system.
+In a Linux environment, for example, the default capacity of a JuiceFS type file system is identified as `1.0P` by using the `df` command that comes with the system.
```

Comment:
Safe to assume that Linux users are familiar with df.

For Linux, the default capacity of a JuiceFS type file system is identified as `1.0P` by using the `df` command.